bin/uglifyjs

```diff
@@ -173,6 +173,10 @@ function convert_ast(fn) {
 }
 
 function run() {
+    if (program.sourceMap && program.sourceMap.content) {
```

Comment:
This won't work for `sourceMap.content = "inline"`

Reply:
Should I add it to the if or should I just take your suggestions above?

Reply:
If you don't mind using https://github.com/mishoo/UglifyJS2/pull/3040#issuecomment-377664963 - the clean-up is very useful since the only purpose for `parse_source_map()` was to handle `sourceMap.content`

Reply:
Okay, thanks